### ISSUE TITLE
Softmax or logsoftmax for auxiliary prediction

### ISSUE THREAD

**Issue by haoyangz** [gitlimlab/ACGAN-PyTorch#3]
The current code uses softmax for auxiliary prediction and during training uses NLL Loss. But according to [here](http://pytorch.org/docs/master/nn.html) it seems that the right last layer to go with NLL is LogSoftmax instead?

**Comment by zhangrong1722** [comment]
However,I found only Softmax rather than LogSoftmax works.Could you verify the differences between these?
